mmcv/runner/checkpoint.py

```diff
@@ -162,8 +161,17 @@ def _load_checkpoint(filename, map_location=None):
         model_name = filename[14:]
         checkpoint = load_url_dist(model_urls[model_name])
     elif filename.startswith('open-mmlab://'):
+        model_urls = get_external_models()
         model_name = filename[13:]
-        checkpoint = load_url_dist(open_mmlab_model_urls[model_name])
+        model_url = model_urls[model_name]
+        # check if is url
+        if model_url.startswith(('http://', 'https://')):
+            checkpoint = load_url_dist(model_url)
+        else:
+            filename = osp.join(_get_mmcv_home(), model_url)
+            if not osp.isfile(filename):
+                raise IOError(f'{filename} is not a checkpoint file')
+            checkpoint = torch.load(filename, map_location=map_location)
```

Comment:
We may consider package the above logic into a method, to support load checkpoint from a url or local file, (and potentially s3).